calico-cloud/users/create-custom-role-for-entra-id-group.mdx

```diff
@@ -0,0 +1,55 @@
+---
+description: Create custom roles for Entra ID groups.
+---
+
+import IconUser from '/img/icons/user-icon.svg';
+
+# Create a custom role for an Entra ID group
+
+If you have Microsoft Entra ID configured as your identity provider, you can assign custom roles to Entra ID (formerly Azure AD) security group.
+This lets you manage role-based access to Calico Cloud directly from your identity provider portal.
+
+## Prerequisites
+
+* You have owner or administrator permissions to the Calico Cloud Manager UI.
+* You have administrator permissions for your organization in the Azure Portal.
+
+## Create and configure an Azure web app
+
+You need to create an Azure web app to securely connect Calico Cloud to your Entra ID security group data.
+
+***Procedure***
+
+1. From the Azure portal, search for `app registrations`. Under **Services**, select **App registration**.
+1. Select **New registration** and complete the form:
+   * Enter a **Name**.
+   * Under **Supported account types**, select **Accounts on this organization directory only**.
+   * Under **Redirect URI**, add a URI with the type **Web** and the value `https://auth.calicocloud.io/login/callback`.
+1. From the Azure portal, search for app registrations and select the app you created.
+1. Under **Manage**, select **Token configuration** and then click **Add groups claim**.
+1. On the **Edit groups claim** panel, select the **security groups** checkbox.
+1. Under **Manage**, select **Authentication**.
+   Under **Implicit grant and hybrid flows** select the **ID tokens** checkbox.
```

Comment:
There's also a "Save" button that must be clicked after selecting "ID tokens". The same applies after selecting "security groups" on the "Edit groups claim panel".

Reply:
I'll add this.